.ci/pipeline_definitions

```diff
@@ -59,7 +59,7 @@ terraformer:
               PROVIDER: slim
     steps:
       verify:
-        image: 'golang:1.16.3'
+        image: 'golang:1.18.5'
```

Comment:
You could also add handling for the old release branches to continue to use `1.16.3`. For an example see the pipeline definition in branch.cfg of gardener/gardener.